src/components/Popups/PopupHighlight.scss

```diff
@@ -0,0 +1,56 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba {
+    $popup-arrow-height: 3px;
+    $popup-arror-width: 4px;
+    $popup-content-bg: $bdl-gray;
+    $popup-icon-margin: 8px;
+
+    .ba-PopupHighlight {
+        pointer-events: auto;
+
+        // Top-facing arrows
+        &[data-popper-placement*='bottom'] {
```

Comment:
Do we know that this will always be top-facing? It doesn't look like we've disabled `preventOverflow` or `flip`?

Reply:
The default setting for bottom-facing is good, don't need to override. It won't be left/right facing.